_schema.graphql

```diff
@@ -5447,10 +5447,6 @@ enum EventStatus {
   RUNNING_AND_UPCOMING
 }
 
-type ExactPrice {
```

Comment:
This ultimately wasn't needed or being used anywhere (besides vortex stitching) so we nixed it.